templates/zerver/help/import-from-slack.md

```diff
@@ -89,66 +79,25 @@ in order to export private message data.
 
 ### Import your data into Zulip
 
-To start using Zulip, you will need to choose between Zulip Cloud and
-self-hosting Zulip. For a simple managed solution, with no setup or maintenance
-overhead, you can [sign up](/new/) for Zulip Cloud with just a few clicks.
-Alternatively, you can [self-host](/self-hosting/) your Zulip organization. See
-[here](/help/zulip-cloud-or-self-hosting) to learn more.
-
-!!! warn ""
-
-    **You can only import a Slack workspace as a new Zulip organization.** Slack
-    workspace history cannot be added into an existing Zulip organization.
+{!import-your-data-into-zulip.md!}
 
 {start_tabs}
 
 {tab|zulip-cloud}
 
-#### Import into a Zulip Cloud organization
-
-{!send-us-info.md!}
-
-1. The subdomain you would like to use for your organization. Your Zulip chat will
-   be hosted at `<subdomain>.zulipchat.com`.
+{!import-into-a-zulip-cloud-organization.md!}
 
-2. The `zip` file containing your Slack message history export.
-
-3. Your Slack **Bot User OAuth Token**, which will be a long
-   string of numbers and characters starting with `xoxb-`
-
-!!! warn ""
-
-    If the organization already exists, the import process will overwrite all data
-    that's already there. If needed, we're happy to preserve your data by moving an
-    organization you've already created to a new subdomain prior to running the import process.
+1. Your Slack **Bot User OAuth Token**, which will be a long
+   string of numbers and characters starting with `xoxb-`.
 
 {tab|self-hosting}
 
-#### Import into a self-hosted Zulip server
+{!import-into-a-self-hosted-zulip-server.md!}
 
-Zulip's Slack import tool is robust, and has been used to import Slack
-workspaces with 10,000 members and millions of messages. If you're planning on
-doing an import much larger than that, or run into performance issues when
-importing, [contact us](/help/contact-support) for help.
+    !!! warn ""
 
-1. Follow steps
-   [1](https://zulip.readthedocs.io/en/stable/production/install.html#step-1-download-the-latest-release)
-   and
-   [2](https://zulip.readthedocs.io/en/stable/production/install.html#step-2-install-zulip)
-   of the guide for [installing a new Zulip
-   server](https://zulip.readthedocs.io/en/stable/production/install.html).
-
-1. Copy the `zip` file containing your Slack message history export onto your Zulip
-server, and put it in `/tmp/`.
-
-1. Log in to a shell on your Zulip server as the `zulip` user.
-
-1. To import into an organization hosted on the root domain
-(`EXTERNAL_HOST`) of the Zulip installation, run the following commands, replacing
-`<token>` with your Slack **Bot User OAuth Token**.
-
-    !!! tip ""
-        The import could take several minutes to run, depending on how much data you're importing.
+        * In the third command, replace `<token>` with your
```

Comment:
Putting this into a warning isn't really working for me; I think the previous phrasing was better.

Reply:
I wanted to reuse Steps 1-4 in the other help articles, but using the previous phrasing and repeating step 4 in the other articles works for me.